### PR TITLE
trimming Particle effects blocks

### PR DESCRIPTION
**Pull request by jwunderl**
closes Microsoft/pxt-arcade#574 (see discussion there)

arcade-screenshot

### CODE REVIEW

**Comment by jwunderl** [comment]
@abchatra wanted it customized as an optional parameter to ``game.over``, not as a separate block

**Comment by jwunderl** [comment]
here's what it'll look like with the win/lose field editor

**Comment by pelikhan** [comment]
Looks great!

File: libs/game/game.ts [hunk 1]
```diff
@@ -111,8 +111,6 @@ namespace game {
      * @param win whether the animation should run on a win (true)
      * @param effect
      */
-    //% group="Gameplay"
-    //% blockId=setGameOverEffect block="set game over effect for win %win=toggleYesNo to %effect"
     export function setGameOverEffect(win: boolean, effect: effects.BackgroundEffect) {
```
Comment (pelikhan) [inline libs/game/game.ts]: We automatically do gameover when life == 0, so we need this block to configure the effect.
https://github.com/Microsoft/pxt-common-packages/pull/634/files#diff-7c3ef1deff0b4e062963ed4e50585d99L116

**Pull request merged**